eth-custodian/contracts/test/NearProverMock.sol

```diff
@@ -1,31 +1,31 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
-pragma solidity ^0.6.12;
+pragma solidity ^0.8;
 
 import 'rainbow-bridge/contracts/eth/nearprover/contracts/INearProver.sol';
 
 contract NearProverMock is INearProver {
     function proveOutcome(
-        bytes memory proofData, 
-        uint64 blockHeight
+        bytes memory, // proofData
+        uint64 // blockHeight
     )
         public
-        view
-        override 
-        returns(bool) 
+        pure
+        override
+        returns(bool)
     {
         return true;
     }
 }
 
 contract NearNegativeProverMock is INearProver {
     function proveOutcome(
```

Comment:
Why we have this function ? if it will be implemented in the future, you can keep the parameter names even if there are compiler warnings for unused parameters.

Reply:
This is a mock Prover that is used in tests. It should always return `true`, it doesn't need any additional implementation in the future.